strr-pm-web/app/pages/platform/application.vue

```diff
@@ -125,6 +125,18 @@ const setPreviousStep = () => {
   }
 }
 
+// something like this? need to discuss options
+// how can we set loading state on the submit and pay button?
+// i wonder if tracking the steps in a store would be useful
```

Comment:
hmm good idea that makes it more intuitive I think. Otherwise we'll need to provide 'loading' as an option in the button config and then we're calling a util function to edit the meta data to toggle it which feels wrong